app/views/layouts/application.html.erb

```diff
@@ -6,9 +6,26 @@
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
   <% if current_user %>
-    <p >
-      <%= link_to "Sign out", destroy_user_session_path, method: :delete %>
-    </p>
+    <div class="container">
```

Comment:
Invalid HTML here, lines 8 to 29 belong inside the `body` tags, not inside the `head` tags.